src/index.ts

```diff
@@ -3,6 +3,7 @@ export * from './context/theme';
 export * from './context/analyticsContext';
 
 export * from './containers/PageConstructor';
+export * from './containers/PageConstructor/components/ConstructorBlocks';
```

Comment:
lets add index.ts file to `PageConstructor/components` with content `export {ConstructorBlocks} from './ConstructorBlocks';`

and then in `src/containers/PageConstructor/index.ts` add line `export * from './components';`